packit_service/worker/handlers/distgit.py

```diff
@@ -405,12 +405,13 @@ def run_for_target(
         )
 
         try:
-            downstream_pr = self.sync_branch(
+            downstream_pr, additional_prs = self.sync_branch(
                 branch=branch,
                 model=sync_release_run_model,
             )
             logger.debug("Downstream PR created successfully.")
             model.set_downstream_pr_url(downstream_pr_url=downstream_pr.url)
+            model.set_fast_forward_prs({branch: pr.url for (branch, pr) in additional_prs.items()})
```

Comment:
I know this is quick. But I am wondering if we should instead save the info in the db as we do for the others prs and mark them as fast forwarded prs. We could have a column or even better a separate table to create the relationship with the parent pr. In this way we could later find all created prs and their relationships easily? 

We can also do this later if necessity arise.

Reply:
I was thinking about this, but wasn't sure if the additional DB complexity is worth introducing this. Let me try it out and we can decide then.

Reply:
pushed a commit with how this would look like, let me know wdyt 🙏